packages/core/src/engines/DragEngine.ts

```diff
@@ -141,14 +141,14 @@ export class DragEngine extends CoordinatesEngine<'drag'> {
 
     if (config.preventScroll && !state._preventScroll) {
       if (state.axis) {
-        if (state.axis === 'x') {
+        if (state.axis === (config.preventScrollAxis || 'x')) {
```

Comment:
I don't think you need `|| 'x'` since `config.preventScrollAxis` should be defaulted to `x`, unless I'm missing something?